.env

```diff
@@ -1,4 +1,4 @@
 SKIP_PREFLIGHT_CHECK=true
-CONSUMER_KEY=
-CONSUMER_SECRET=
-CALLBACK_URL=
\ No newline at end of file
+CONSUMER_KEY="1IsryRNkS3BjSKRfY6LqlJIO8"
```

Comment:
👀 We should remove these.

Reply:
Consumer key has changed since then, so we're all good!